client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import handlers from './wpcom';
+
+export const middleware = store => next => action =>
+	has( handlers, action.type )
+		? handlers[ action.type ].forEach( handler => handler( store, action ) )
```

Comment:
Could we provide a convenience for setting a single handler for an action type to not need to pass it in an array? In other words:

```js
toArray( handlers[ action.type ] ).forEach( handler => handler( store, action ) )
```

Seems like single handler would be the most common case, no?